docs/DeveloperGuide.md

```diff
@@ -97,7 +103,7 @@ Feature: the name of my feature
       | null |
 ```
 
-More examples can be seen in the [generator Cucumber features](https://github.com/finos/datahelix/tree/master/orchestrator/src/test/java/com/scottlogic/datahelix/generator/orchestrator/cucumber). An outline of how Cucumber is used within DataHelix can be found [here](./developer/CucumberCookbook.md).
+More examples can be seen in the [generator Cucumber features](https://github.com/finos/datahelix/tree/master/orchestrator/src/test/java/com/scottlogic/datahelix/generator/orchestrator/cucumber). This document shows [an outline of how Cucumber is used within DataHelix](./developer/CucumberCookbook.md).
```

Comment:
Again, for me `This document` is the reference and should be in the link

Reply:
Link text updated slightly